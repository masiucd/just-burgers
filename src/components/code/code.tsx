import styled from "styled-components";

const Code = styled.code`
  display: inline-block;
  color: ${(props) => props.theme.colors.darkblue};
  font-size: 1.2rem;
  background: #f5f5f5;
  margin-top: 1rem;
  border-radius: 5px;
  padding: 0.5rem;
  @media (max-width: 800px) {
    font-size: 1rem;
  }
  ::before {
    content: "$ ";
  }
`;

export { Code };
