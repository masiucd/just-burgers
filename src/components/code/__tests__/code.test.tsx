import React from "react";
import { Code } from "../code";
import { render } from "@testing-library/react";

it("matches snapshot", () => {
  const { asFragment } = render(<Code>const gatsby = "awesome"</Code>);
  expect(asFragment()).toMatchSnapshot();
});
