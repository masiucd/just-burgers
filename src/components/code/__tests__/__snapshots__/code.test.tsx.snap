// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <code
    class="sc-bdVaJa QhKbv"
  >
    const gatsby = "awesome"
  </code>
</DocumentFragment>
`;
